db/seeds.rb

```diff
@@ -1,8 +1,30 @@
-# frozen_string_literal: true
+# frozen_string_literal= true
```

Comment:
I'm pretty sure this change wasn't intentional. It should be reverted

Reply:
Yes